Sources/UI/Experiences/Models/Screen.swift

```diff
@@ -72,7 +72,7 @@ public struct Screen: Decodable {
 // MARK: AttributeRepresentable
 
 extension Screen: AttributeRepresentable {
```

Comment:
`AttributeRepresentable` has been removed in this PR. You'll need to update this to match the approach you took in the `Beacon` and `Geofence` classes.